src/shared/libmount-util.c

```diff
@@ -38,3 +38,23 @@ int libmount_parse(
         *ret_iter = TAKE_PTR(iter);
         return 0;
 }
+
+int libmount_is_leaf(
+                struct libmnt_table *table,
+                struct libmnt_fs *fs) {
+        int r;
+
+        _cleanup_(mnt_free_iterp) struct libmnt_iter *iter_children = NULL;
+        iter_children = mnt_new_iter(MNT_ITER_FORWARD);
+
+        if (!iter_children)
```

Comment:
please drop the empty line here. we generally have the rule to keep function calls and their immediate error handling together. hence maybe put an empty line before the line with the mnt_new_iter() call, but not immediately after, as the `if` here is then the error handling